example/android/build.gradle

```
@@ -1,13 +1,10 @@
 buildscript {
     ext {
         // Android Installation Step 1a: Set compatible minSdkVersion and compileSdkVersion
-        minSdkVersion = 23 // Minimum supported by Klaviyo Android SDK
-        targetSdkVersion = 33 // Typically minSdkVersion <= targetSdkVersion <= compileSdkVersion
-        compileSdkVersion = 34 // Klaviyo Android SDK is compiled against 34
+        minSdkVersion = 24 // Minimum supported by Klaviyo Android SDK
+        targetSdkVersion = 35 // Typically minSdkVersion <= targetSdkVersion <= compileSdkVersion
+        compileSdkVersion = 35 // Klaviyo Android SDK is compiled against 35
```

This comment can be removed, when i added it, RN supported something like 21+, but we had to narrow to 23+ bc our android SDK has min version of 23. Now its kinda the reverse since RN has bumped up their min supported android version.
And I think it is ok to just compile against latest. I should bump that up on Android SDK too. 
```suggestion
        minSdkVersion = 24
        targetSdkVersion = 35 // Typically minSdkVersion <= targetSdkVersion <= compileSdkVersion
        compileSdkVersion = 35
```